### DISCUSSION TITLE
Add affected projects in New vulnerability mail template

### DISCUSSION THREAD

**Discussion by rkg-mm** [DependencyTrack/dependency-track#1437] [Q&A]
Hi,
is it possible to have the affected projects in the mail template for new vulnerabilities? Currently it's only part of new vulnerable dependency template (though, i never got this mail yet? not sure when its triggered), but not in the mail coming when new vulnerabilities are found.

Thanks!

**Comment by stevespringett** [comment]
Yes, I think its possible. PRs are welcome
